src/olympia/migrations/907-drop-password-and-confirmationcode.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE users
+    DROP COLUMN `password`,
+    DROP COLUMN `confirmationcode`;
```

Comment:
In theory just a few seconds, I don't have too much data locally to verify unfortunately. I'd take a closer look on `-dev` and `-stage` to make sure it stays "a few seconds".